### PR TITLE
feat: Add 9n7d test case

### PR DESCRIPTION
**Pull request by kdheepak**
This PR adds a 9n7d test case that can be solved by HiGHS in a few seconds.

Here's an example of running it:

image

### CODE REVIEW

File: openTEPES/9n/oT_Data_RESEnergy_9n.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-﻿,,RESEnergy
+,,RESEnergy
```
Comment (kdheepak) [inline openTEPES/9n/oT_Data_RESEnergy_9n.csv:1]: There appears there's a Byte Order Mark character at the beginning of the file. 

image

Related: https://stackoverflow.com/a/60517948/5451769

I recommend removing these characters as they can cause issues with parsing.

**Comment by kdheepak** [comment]
I redid this PR without adding any new files. This PR now modifies the 9n folder, runs the test and then undoes the modification.

**Review by erikfilias — APPROVED** [review]
Thank you, @kdheepak; I will merge it. Testing the model by including the solving phase and using Highs is excellent.

**Comment by kdheepak** [comment]
Just to clarify, this test doesn't pass consistently for me. However, it'll be good to get it to a point where it does pass consistently.

**Pull request merged**